src/bot.ts

```diff
@@ -18,6 +18,82 @@ import { is, not } from './utils'
 export const opencollective = (app: probot.Application): void => {
   app.log('OpenCollective Bot up!')
 
+  app.on('installation_repositories', async (context: probot.Context) => {
+    //get 'added' action as condition before adding the config file
+    if (context.payload.action === 'added') {
+      let added_repos = context.payload.repositories_added
+
+      // This can open PRs with new files at any point in time, but this
+      // gets called when the app is first installed on a repo
+      context.log({ event: context.event, action: context.payload.action })
+
+      let fields = {
+        file: {
+          path: '.github/opencollective.yml',
+          content: `collective: graphql-shield
+        tiers:
+          - tiers: '*'
+            labels: ['backer']
+            message: 'Hey <link>'
+          - tiers: ['Sponsor']
+            labels: ['sponsor']
+            message: 'Hey sponsor <link>'
+        invitation: |
+          Hey <author> :wave:,
+          This is an optional message to your audience. Check the
+          default message below.
+        
+          <link>`,
+        },
+        pr: {
+          body: 'The body of your Pull request',
+          title: 'The title of your Pull Request',
+        },
+      }
+
+      const branch = `add-${fields.file.path}` // your branch's name
+      const content = Buffer.from(fields.file.content).toString('base64') // content for your configuration file
+
+      const reference = await context.github.git.getRef({
+        owner: context.repo.owner,
+        repo: context.repo.repo,
+        ref: 'heads/master',
+      }) // get the reference for the master branch
+
+      console.log(
```

Comment:
Don't use that pattern with console.log, make it clear what you are logging. When possible, single line message.

Reply:
Alright, will update.